src/jit/compiler.h

```diff
@@ -1771,7 +1773,11 @@ class Compiler
     // a PSPSym for functions with any EH.
     bool ehNeedsPSPSym() const
     {
+#ifdef _TARGET_X86_
```

Comment:
I currently have no evidence whether Linux/x86 unwider can guarantee the correctness of EBP. We need more investigation. 

In addition, I encountered the following assertion failure while codegen once I tried to push PSPSym,:
```
Assert failure(PID 2080 [0x00000820], Thread: 2080 [0x0820]): Assertion failed 'u2.emitArgTrackTop <= u2.emitArgTrackTab + emitMaxStackDepth' in 'System.AppDomain:SetupDomain(bool,ref,ref,ref,ref):this' (IL size 72)

    File: /home/parjong/projects/dotnet/coreclr/src/jit/emit.cpp Line: 6819
    Image: /root/dotnet-test/overlay/current/corerun
```

It would be better to enable PSPSym-related features in a separate PR.